### DISCUSSION TITLE
Fix tailwind configuration

### DISCUSSION THREAD

**Discussion by mayank1513** [vercel/turborepo#6714] [Help] [answered]
Hey friends,

I was just trying to add a tailwind example for one of my packages. However, it seems that I am making some silly mistake here. It doesn't seem to apply tailwind styles.

Here's the code - https://github.com/react18-tools/nextjs-themes/tree/tailwind/examples/tailwind

Any help / suggestions welcomed

**Comment by mayank1513** [comment]
Is this a bug?

**Comment by anthonyshew** [comment, marked as answer]
Answered on your issue here: https://github.com/vercel/turbo/issues/6717#issuecomment-1843925997

In the future, please only open one entity for your question/issue. Helps us keep the repo cleaned up!
